build.yaml

```diff
@@ -3551,10 +3501,6 @@ steps:
         namespace:
           valueFrom: default_ns.name
         mountPath: /user-tokens
-      - name: worker-deploy-config
```

Comment:
Do you need to set `HAIL_DEFAULT_NAMESPACE` for this job or was the secret not necessary?

Reply:
The namespace is already specified in the command: `hailctl curl {{ default_ns.name }}`